de.hoppfoundation.klassenzimmer.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id type="desktop">de.hoppfoundation.klassenzimmer</id>
+    <launchable type="desktop-id">de.hoppfoundation.klassenzimmer.desktop</launchable>
+    <name>Digitales Klassenzimmer</name>
+    <summary>Digitales Klassenzimmer - Videokonferenztool Jitsi</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>Apache-2.0</project_license>
+    <developer_name>Hopp Foundation for Computer Literacy &amp; Informatics gGmbH</developer_name>
+    <url type="homepage">https://www.hopp-foundation.de/schule/unterstuetzung/videokonferenztool-jitsi/</url>
+    <description>
+        <p>Wir stellen diese auf Jitsi basierende App unseren Kooperationsschulen in der Metropolregion Rhein Neckar zur Verfügung. Die App ermöglicht die Teilnahme an virtuellen Klassenzimmern auf unseren in Deutschland stehenden Servern. Sie ist frei von Trackern und stellt eine Alternative zum Browser (bspw. Chromium) dar.</p>
+	<p>Auch für andere Konferenzsysteme ist diese App nutzbar. Dies erfordert dann eine entsprechende Änderung in den Server-Einstellungen.</p>
+        <p>Bei Fragen und Problemen schreiben Sie uns an jitsi@hopp-foundation.de!</p>
```

Comment:
We usually expect the description to be in English. IIRC you can do something along `<description xml:lang="de">` to translate it and it will be displayed in native language in GNOME Software and KDE Discover.

Reply:
@barthalion Thank you for the suggestion, I have added the English metadata and marked the German as suggested. Based on https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html and some examples from gnome on my system at /usr/share/appdata I have translated it paragraph by paragraph.